csharp/Facebook.Yoga.Desktop.sln

```diff
@@ -0,0 +1,102 @@
+﻿
```

Comment:
I know nothing about BUCK, but can't it invoke arbitrary command line tools? xbuild/msbuild?